BackEnd/Core/sailfish-common/src/main/java/com/exactpro/sf/common/impl/messages/all/configuration/sample/Dictionary.java

```diff
@@ -0,0 +1,345 @@
+        private final static long serialVersionUID = -1L;
+        @XmlElement(name = "field", type = Field.class)
+        protected List<IField> fields;
+
+        /**
+         * Gets the value of the fields property.
+         *
+         * <p>
+         * This accessor method returns a reference to the live list,
+         * not a snapshot. Therefore any modification you make to the
+         * returned list will be present inside the JAXB object.
+         * This is why there is not a <CODE>set</CODE> method for the fields property.
+         *
+         * <p>
+         * For example, to add a new item, do as follows:
+         * <pre>
+         *    getFields().add(newItem);
+         * </pre>
+         *
+         *
+         * <p>
+         * Objects of the following type(s) are allowed in the list
+         * {@link Field }
+         *
+         *
+         */
+        public List<IField> getFields() {
+            if (fields == null) {
+                fields = new ArrayList<>();
+            }
+            return this.fields;
+        }
+
+    }
+
+
+    /**
+     * <p>Java class for anonymous complex type.
+     *
+     * <p>The following schema fragment specifies the expected content contained within this class.
+     *
+     * <pre>
+     * &lt;complexType>
+     *   &lt;complexContent>
+     *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+     *       &lt;sequence>
+     *         &lt;element name="message" type="{http://exactprosystems.com/dictionary}Message" maxOccurs="unbounded" minOccurs="0"/>
+     *       &lt;/sequence>
+     *     &lt;/restriction>
+     *   &lt;/complexContent>
+     * &lt;/complexType>
+     * </pre>
+     *
+     *
+     */
+    @XmlAccessorType(XmlAccessType.FIELD)
+    @XmlType(name = "", propOrder = {
+        "messages"
+    })
+    public static class Messages implements Serializable
+    {
+
+        private final static long serialVersionUID = -1L;
+        @XmlElement(name = "message", type = Message.class)
+        protected List<IMessage> messages;
+
+        /**
+         * Gets the value of the messages property.
+         *
+         * <p>
+         * This accessor method returns a reference to the live list,
+         * not a snapshot. Therefore any modification you make to the
+         * returned list will be present inside the JAXB object.
+         * This is why there is not a <CODE>set</CODE> method for the messages property.
+         *
+         * <p>
+         * For example, to add a new item, do as follows:
+         * <pre>
+         *    getMessages().add(newItem);
+         * </pre>
+         *
+         *
+         * <p>
+         * Objects of the following type(s) are allowed in the list
+         * {@link Message }
+         *
+         *
+         */
+        public List<IMessage> getMessages() {
+            if (messages == null) {
+                messages = new ArrayList<>();
+            }
+            return this.messages;
+        }
+
+    }
+
```

Comment:
These classes should be removed and replaced by List and some annotations to achieve the same result